src/Microsoft.AspNetCore.WebHooks.Receivers.AzureAlert/AzureAlertCondition.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNetCore.WebHooks
```

Comment:
Recommend ignoring the new C# files. They are identical to the Microsoft.AspNet.WebHooks versions except for their namespace and (in a couple of cases) `using`s or doc comments. That is, the edits are the opposite to those above except the VS suggestions were identical.

Reply:
OK, I believe you